.github/workflows/push.yml

```
@@ -0,0 +1,44 @@
+name: Projects check
+
+on:
+  push:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  projectsbuild:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: kpsr-sdk
+    name: Check ${{ matrix.name }}
+    runs-on: [self-hosted, general, X64]
```

```suggestion
    runs-on: ubuntu-latest
```
Public repositories use github public runners. Private ones are not availabe here. Thats why CI stuck for this PR